src/flatc.cpp

```diff
@@ -607,7 +609,12 @@ int FlatCompiler::Compile(int argc, const char **argv) {
       } else if (arg == "--json-nested-bytes") {
         opts.json_nested_legacy_flatbuffers = true;
       } else if (arg == "--ts-flat-files") {
-        opts.ts_flat_file = true;
+        opts.ts_flat_files = true;
+        opts.ts_entry_points = true;
+        opts.generate_all = true;
```

Comment:
I don't think I have an opinion on this, and since I may be the only user of --ts-flat-files currently, it's probably a non-issue. But I do want to note that my original intent here was not to imply `--gen-all` with `--ts-flat-files`.

Reply:
It was intentional because when generating a bundle all module imports must be resolvable.

Reply:
I noticed the name went from `ts_flat_file` to `ts_flat_files`, did we keep the flag name the same though?

Reply:
Yes flag name was and is `--ts-flat-files` (https://github.com/google/flatbuffers/blob/e1a2f688e0d12385fd977cb45d58453d88d7c0e2/src/flatc.cpp#L609)